Interface.H

```diff
@@ -37,7 +37,8 @@ protected:
     int numDataLocations_ = 0;
 
     //- Vertex IDs assigned by preCICE
-    int * vertexIDs_;
+    int * centerIDs_;
+    int * nodeIDs_;
```

Comment:
`nodeIDs` is not used anywhere else. Therefore, remove it and keep the name `vertexIDs`.

Reply:
Correct, this is deprecated.